components/ArticlePageHeader.vue

```diff
@@ -0,0 +1,213 @@
+<script setup lang="ts">
+  import VFlexibleLink from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VFlexibleLink.vue'
+import VShareTools from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VShareTools.vue'
+import VShareToolsItem from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VShareToolsItem.vue'
+  const { $analytics } = useNuxtApp()
+  
+  const props = defineProps({
+    title: {
+      type: String,
+      default: ''
+    },
+    progress: {
+      type: Number,
+      default: 0
+    },
+    donateUrlBase: {
+      type: String,
+      required: true,
+    },
+    utmCampaign: {
+      type: String,
+      required: true,
+    },
+    shareUrl: {
+      type: String,
+      default: ''
+    },
+    shareTitle: {
+      type: String,
+      default: ''
+    },
+  })
+  const sidebarIsOpen = useSidebarIsOpen()
+  const progressPercentage = computed(() => `${props.progress}%`)
+  const openSidebar = () => { sidebarIsOpen.value = true }
+</script>
+
+<template>
+<header class="article-page-header">
+  <div class="article-page-header-progress">
+    <div class="article-page-header-contents p-3 md:px-5 flex justify-content-between">
+      <div class="article-page-header-left">
+        <v-flexible-link to="/" raw>
+          <LogoGothamist class="article-page-header-logo" />
+        </v-flexible-link>
+        <div class="article-page-header-tagline hidden">News for New Yorkers</div>
+        <div class="hidden">{{props.title}}</div>
+      </div>
+      <div class="article-page-header-center hidden md:block px-4">
+        <span class="article-page-header-title">{{title}}</span>
```

Comment:
- add `h6` class to the parent div 
`<div class="article-page-header-center h6 hidden md:block px-4">`

- remove <span> and just leave the `{{title}}`

Reply:
actually should this be `props.title`?

Reply:
I don't think you have to specify if you don't have a var conflict... but I always put props.var for clarity

Reply:
ah right, you don't need props in the template, just in scripts